package.json

```diff
@@ -38,7 +38,7 @@
     "get-next-version": "node scripts/get-next-version.js",
     "postinstall": "patch-package && ./scripts/run-if-not-ci.sh yarn-deduplicate --strategy=highest && ./scripts/run-if-not-ci.sh yarn build",
     "jscodeshift": "jscodeshift -t ./node_modules/js-codemod/transforms/arrow-function-arguments.js",
-    "lint": "eslint --ext .js,.jsx,.ts,.tsx,.json .",
+    "lint": "eslint --ext .js,.jsx,.ts,.tsx,.json . && yarn lerna exec yarn lint --scope @tooling/system-tests",
```

Comment:
To me, it'd make more sense if this ran in the `type-check` script.

Reply:
[a04a6b6](https://github.com/cypress-io/cypress/pull/20430/commits/a04a6b66f1348e421fec578197f7a88bf8ee5ca5)